src/mixins/name-request-mixin.ts

```diff
@@ -17,7 +17,7 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
   generateNameRequestState (nr: any, filingId: number): NameRequestIF {
     return {
       nrNumber: nr.nrNum,
-      entityType: this.nrTypeToEntityType(nr),
+      entityType: nr.legalType,
```

Comment:
Vysakh assures me that all NRs fetched via Legal API will have this value, and it will already have been correctly converted from NR type (eg, BC -> BEN).

Reply:
This is a nice change! 

I'm not aware of the full scope of the api change but will this change require some attention in Auth as well? 
I know we've had to do similar conversions with the entity table and creating draft filings etc, i think*